CloudinaryManager.js

```diff
@@ -0,0 +1,58 @@
+export default class CloudinaryManager {
+    constructor(options) {
+        this.options = options;
+    }
+
+    createMediaString(formdata) {
+        var mediaStr;
```

Comment:
remove if else and use ternary instead.
`return formdata.minWidth ? `media...${formdata.minWidth}` : `media...${formdata.maxWidth}``

Reply:
fixed